### DISCUSSION TITLE
My Profile isn't appear in Google or from another Machine

### DISCUSSION THREAD

**Discussion by amirmehrez** [community/community#76281] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello, 
I have a Problem with my github Profile, it didn't appear from another Machine or my repositories too, First the problem did beginning with the GitHub Pages but actually my Profile and all my repos aren't show.
I have all my configuration in Public and my repos too.
Help me please

**Comment by amirmehrez** [comment, marked as answer]
The problem was with the new option for show archivements of the private repositories

**Reply by Boldi26** [reply]
I have the same problem, how did you solve the issue?

**Reply by rohithsomella** [reply]
its not an complete solution for the problem even my profile is not visible in other systems though i give them the link with username in it. and how many time i do add new readme file with the same username still it is not appearing on my profile its just went into a new repo thats it. what to do ??

**Reply by Grey-Rabbit002** [reply]
what is the solution ??

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @226218! We're happy you're here. I'm happy that you were able to find the answer to your question but, for future reference, I do want to note that the Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this particular post to the correct category for you. Cheers!

**Reply by DebiduttaPanigrahi** [reply]
Hello,
I have a Problem with my github Profile, it didn't appear from another Machine or my repositories too, First the problem did beginning with the GitHub Pages but actually my Profile and all my repos aren't show.
I have all my configuration in Public and my repos too.
Help me please
Account name is DUTTA-78150

**Comment by bleuubuntu** [comment]
There is no answer for this problem or what?

**Comment by Sinakhx** [comment]
I have the same issue. no solution worked.

**Comment by onliveserver** [comment]
I also have the same problem, my profile URL is not showing anywhere, this is the URL - https://github.com/onliveserver.
I checked all my settings but it still doesn't show.

**Reply by Diomedes-Mist** [reply]
How did you fix this issue?
